superset-frontend/src/CRUD/CollectionTable.tsx

```diff
@@ -27,6 +27,8 @@ import './crud.less';
 interface CRUDCollectionProps {
   allowAddItem?: boolean;
   allowDeletes?: boolean;
+  allowSyncFromSource?: boolean;
+  onSyncFromSource?: () => void;
```

Comment:
Both props seem to be `Datasource` table specific and should not be added to the general `CollectionTable` component.

Reply:
I get your point about it being specific to the Datasource only though. Let me check if I can move it out while keeping the same position.